install-helpers-deps/action.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I think this might read more naturally as `install-helper-deps` or even, more explicitly, as `install-test-helper-deps`.

Reply:
true, choosing later